spring-kafka-docs/src/main/antora/modules/ROOT/pages/testing.adoc

```diff
@@ -350,6 +354,38 @@ They include:
 * xref:testing.adoc#kafka-testing-junit4-class-rule[JUnit4 Class Rule]
 * xref:testing.adoc#kafka-testing-embeddedkafka-annotation[`@EmbeddedKafka` Annotation or `EmbeddedKafkaBroker` Bean]
 
+[[kafka-testing-junit4-embedded-broker]]
+=== Junit4 Embedded Broker
+
+The following example shows how to create an embedded broker in Junit4:
+[source, java]
+----
+@SpringBootTest
+public class MyApplicationTests {
+
+    @Autowired
+    private final EmbeddedKafkaBroker broker;
+
+    @Autowired
+    private KafkaTemplate<String, String> template;
+
+    @Test
+    public void test() {
+        ...
+    }
+
+    @Configuration
+    public static class MyConfiguration {
+        @Bean
+        public EmbeddedKafkaBroker embeddedKafkaBroker() {
+            return new EmbeddedKafkaKraftBroker(1, 1, "someTopic");
+        }
+
+    }
+
+}
+----
+
 [[kafka-testing-junit4-class-rule]]
 === JUnit4 Class Rule
```

Comment:
I would prefer to have all the JUnit 4 text removed in favor of a single sentence that its support is deprecated for removal in the next minor version.